ts/pages/staking/staking_pool.tsx

```diff
@@ -240,43 +246,58 @@ const tradingPairs = [
 export const StakingPool: React.FC<StakingPoolProps & RouteChildrenProps> = props => {
     const { poolId } = useParams();
 
+    const networkId = useSelector((state: State) => state.networkId);
+    const apiClient = useAPIClient(networkId);
+    const [stakingPool, setStakingPool] = useState<PoolWithHistoricalStats | undefined>(undefined);
+
+    useEffect(() => {
+        apiClient.getStakingPoolByIdAsync(poolId)
+            .then(res => setStakingPool(res.stakingPool))
+            .catch(e => console.error(e));
+    }, []);
+
+
     // Ensure poolId exists else redirect back to home page
     if (!poolId) {
         return (
             <Redirect to={WebsitePaths.Staking}/>
         );
     }
 
+    if (!stakingPool) {
+        return null;
+    }
+    const currentEpoch = _.last(stakingPool.epochRewards);
+
     return (
         <StakingPageLayout isHome={true} title="Staking pool">
             <DashboardHero
-                onButtonClick={() => props.history.push(`${WebsitePaths.StakingWizard}?${stringify({ poolId })}`)}
-                title="Binance Staking Pool"
-                websiteUrl="mywebsite.com"
-                poolId="0x1234...1234"
-                isVerified={true}
-                estimatedStake={75}
-                rewardsShared={74}
-                iconUrl=""
+                title={stakingPool.metaData.name}
+                websiteUrl={stakingPool.metaData.websiteUrl}
+                poolId={utils.getAddressBeginAndEnd(stakingPool.operatorAddress)}
+                isVerified={stakingPool.metaData.isVerified}
+                estimatedStake={75} // what's this
+                rewardsShared={74} // what's this
```

Comment:
what are these two values?

Reply:
rewards shared I'm guessing is just user rewards / total rewards, but not sure what `estimatedStake` is